custom_registration/models.py

```diff
@@ -14,7 +14,7 @@
 
 from studygroups.models import Profile
 
-def create_user(email, first_name, last_name, password, mailing_list_signup):
+def create_user(email, first_name, last_name, password, mailing_list_signup, reason_for_registration):
```

Comment:
Please also update the corresponding test cases. You can see the ones failing on [TravisCI](https://travis-ci.org/p2pu/learning-circles/builds/420837167).